### DISCUSSION TITLE
⭐️Star exchange💫

### DISCUSSION THREAD

**Discussion by LeleDallas** [drknzz/GitHub-Achievements#130] [Q&A] [answered]
Hey guys, I wanted to share with you a project I've been working on that I'm really excited about. It's a digital version of the popular game `Cards Against Humanity`.

You can find the GitHub repository for the game here: [Cards Against Humanity Game](https://github.com/LeleDallas/Cards-Against-Humanity)
If you have a moment, I'd be thrilled if you could take a look at the repository and leave a star ⭐️.

Thank you so much for your time, and I hope you have a blast playing the game!

**Comment by VirgilZaccomer** [comment]
Hi please accept this as answer for galaxy brain achievment

**Comment by Manz03b** [comment]
Back me up for the galaxy brain

**Comment by laurentiucozma12** [comment]
I gave you a star, can you give me a star please [Epic Game News](https://github.com/laurentiucozma12/epicgamenews-presentation) ?
Also could I get this marked as the answer?

**Reply by pu-raihan** [reply]
[Cambuzz](https://github.com/pu-raihan/CamBuzz)
check out my project and please leave a **star**

**Comment by laurentiucozma12** [comment]
Also could I get this marked as the answer too? To get discussion achievement for 2 accepted answers.

**Comment by shreya-oeo** [comment]
https://github.com/aulolua/chatroom btw

**Comment by sheeehy** [comment]
https://github.com/sheeehy/Geo-Git-v1  <3

**Comment by HarshDekate** [comment, marked as answer]
Hello! Thank you for sharing your exciting project. We'll definitely check out the GitHub repository for the Cards Against Humanity game and leave a star ⭐️. Best of luck with your project, and we look forward to having fun playing it! 😄🎉
